src/components/dls/Forms/Combobox/Combobox.module.scss

```diff
@@ -0,0 +1,179 @@
+.label {
+  padding: var(--spacing-micro) 0;
+}
+
+.container {
+  justify-content: space-between;
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+.searchInputContainer {
+  text-align: left;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+  height: 100%;
+  border-radius: var(--border-radius-default);
+  border: 1px solid var(--color-borders-hairline);
+  font-size: inherit;
+  cursor: text;
+  outline: 0;
+  padding: 0 var(--spacing-mega);
+  background: var(--color-background-default);
+  color: var(--color-text-default);
+}
+
+.activeInput {
+  border: 1px solid var(--color-primary-medium);
+}
+.searchInputContainer:not(.disabledInput):not(.hasError) {
+  &:hover {
+    border: 1px solid var(--color-primary-medium);
+  }
+}
+
+.hasError {
+  border: 1px solid var(--color-error-medium);
+}
+
+.itemsContainer {
+  border-radius: var(--border-radius-default);
+  overflow: hidden;
+  max-height: calc(5 * var(--spacing-mega));
+  overflow-y: scroll;
+}
+
+.searchIconContainer {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  color: var(--color-alert-medium);
+  pointer-events: none;
+  position: absolute;
+  left: var(--spacing-small);
+  top: 50%;
+  z-index: 1;
+  transform: translateY(-50%);
+  width: var(--spacing-medium);
+  height: var(--spacing-medium);
+}
+
+.clearIconContainer {
+  user-select: none;
+  background: none;
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--spacing-small);
+  height: var(--spacing-small);
+  color: var(--color-alert-medium);
+  position: absolute;
+  right: var(--spacing-medium);
+  top: 50%;
+  transform: translateY(-50%);
+  outline: none;
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+  text-decoration: none;
+}
+
+.searchInput {
+  width: 100%;
+  &:focus {
+    outline: none;
+  }
+}
+
+.comboboxContainer {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  max-width: 100%;
+  width: calc(8 * var(--spacing-mega));
```

Comment:
you can define this as a variable if you'd like (e.g. `$combo-box-width: 3 rem`) . Up to you.